pkg/defaults/policies/files/exec-iptables-root.json

```
@@ -13,6 +13,34 @@
   ],
   "eventSource": "DEPLOYMENT_EVENT",
   "exclusions": [
+    {
+      "name": "Don't alert on deployment haproxy-control-plane-* in namespace openshift-kni-infra",
+      "deployment": {
+        "name": "haproxy-control-plane-*",
+        "scope": {
+          "namespace": "openshift-kni-infra"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment keepalived-control-plane-* in namespace openshift-kni-infra",
+      "deployment": {
+        "name": "keepalived-control-plane-*",
+        "scope": {
+          "namespace": "openshift-kni-infra"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment keepalived-worker-* in namespace openshift-kni-infra",
+      "deployment": {
+        "name": "keepalived-worker-*",
+        "scope": {
+          "namespace": "openshift-kni-infra"
+        }
+      }
+    }
+  ,
```

This is one of your lint errors and would need to be put in the migration/after as well.

```suggestion
    },
```